### DISCUSSION TITLE
Suggestion: Dynamic sun position

### DISCUSSION THREAD

**Discussion by aosfan** [piqueserver/piqueserver#638] [Feature Requests]
It would be great if there is a possibility to add dynamic or at least changeable sun position.
Imagine: battle on a sunset, with long shadows, or even at night, with tracers through dark fog and flashlights.

**Comment by noway** [comment]
Aos protocol only supports sky color. It's a very atmospheric idea though!

**Comment by NotAFile** [comment]
requires #66 which is a lot of work. Once we get that one done we could add as many new packets as we want.
